test/info/info-output.test.js

```diff
@@ -18,7 +18,7 @@ describe('basic info usage', () => {
     });
 
     it('gets info as json', async () => {
-        const { stdout, stderr } = await runInfo(['--output-json']);
+        const { stdout, stderr } = await runInfo(['--output="json"']);
         expect(stdout).toContain('"System":');
         expect(stderr).toHaveLength(0);
```

Comment:
Please add test what output is valid JSON - `JSON.parse` should not throw an error and contains basic fields

Reply:
JSON parse test exist already ( Just below were you have made comment. )
Would add tests for basic fields.